Orange/widgets/gui.py

```diff
@@ -1545,21 +1549,17 @@ def minimumSizeHint(self):
         return sh
 
 
-# TODO comboBox looks overly complicated:
-# - can valueType be anything else than str?
-# - sendSelectedValue is not a great name
 def comboBox(widget, master, value, box=None, label=None, labelWidth=None,
              orientation=Qt.Vertical, items=(), callback=None,
-             sendSelectedValue=False, valueType=str,
```

Comment:
I didn't find anybody specifying a `valueType` other than `str`. This commit removes this argument from calls (in core widgets). If any other widget specifies `valueType`, it will now be silently ignored in `misc`.